packages/common/src/schemas/setting/feature-flag-setting.schema.ts

```diff
@@ -44,7 +44,10 @@ export const featureFlagSettingSchema = Type.Object(
       'ir.client.menu.social',
       'ir.client.menu.emote',
       'ir.client.menu.avaturn',
-      'ir.client.menu.readyPlayerMe'
+      'ir.client.menu.readyPlayerMe',
+      'ir.mt.menu.myDomains',
```

Comment:
We should make this just an untyped string. We don't want MT concepts in the core engine.

Reply:
not sure about this one,  also the FeatureFlagsState lives in the core engine so how the projects can use that feature knowing the feature flag name ? thoughts  @hanzlamateen ? 

other way would be for example `ir-client.menu.myDomains` but i was assuming that we will have a Admin dashboard for features flags for each project and the way to identified is by the structure of the feature flag name like `ir-${project}-flag`

Reply:
i already untyped the flagName to decouple flag names from the core engine, just leaving a generic in the actions of the state to remain type safety in the project context